bench/entt/optimized/sparse_set.cpp.ll

```diff
@@ -230217,7 +230217,7 @@ _ZNK4entt16basic_sparse_setINS_6entityESaIS1_EE7currentES1_.exit: ; preds = %lan
   store i16 %conv2.i.i, ptr %ref.tmp8, align 2, !tbaa !2389
   call void @llvm.lifetime.start.p0(i64 4, ptr nonnull %ref.tmp10) #23
   store i32 0, ptr %ref.tmp10, align 4, !tbaa !889
-  %cmp.i.i384 = icmp eq i16 %conv2.i.i, 0
+  %cmp.i.i384 = icmp ult i32 %8, 1048576
```

Comment:
Regression.

Reply:
Duplicate of https://github.com/dtcxzyw/llvm-opt-benchmark/pull/600#discussion_r1597405950